manifests/example/target-pvc.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: "target-pvc"
+  namespace: "target-ns"
+  labels:
+    app: Host-Assisted-Cloning
+  annotations:
+    k8s.io/CloneRequest: "source-ns/golden-pvc"
```

Comment:
maybe this anno is changed to follow cdi standard of _cdi.kubevirt.io/_?

Reply:
Nope.  See previous comment.